src/vivarium/interface/interactive.py

```diff
@@ -18,18 +18,15 @@
 
 """
 from math import ceil
-from typing import Mapping, List, Callable, Dict, Any
-import warnings
+from typing import List, Callable, Dict, Any
 
 import pandas as pd
 
-from vivarium.framework.configuration import build_simulation_configuration, build_model_specification, ConfigTree
 from vivarium.framework.engine import SimulationContext
-from vivarium.framework.plugins import PluginManager
 from vivarium.framework.time import Timedelta, Time
 from vivarium.framework.values import Pipeline
 
-from .utilities import run_from_ipython, log_progress, raise_if_not_setup
```

Comment:
Replacing the raise_if_not_setup behavior with lifecycle restrictions in the next pr.